docs-team/2018-05/may-10.md

```diff
@@ -1,90 +1,47 @@
-# Learn Team Agenda 2018-05-17
```

Comment:
These notes were saved under the wrong file, so moved them to may-17, hence the diff here.